diff --git a/browser/src/control/Control.Notebookbar.js b/browser/src/control/Control.Notebookbar.js
index 1abe340..3b8de53 100644
--- a/browser/src/control/Control.Notebookbar.js
+++ b/browser/src/control/Control.Notebookbar.js
@@ -593,7 +593,7 @@ window.L.Control.Notebookbar = window.L.Control.extend({
 
 		const docType = this._map.getDocType();
 
-		if (docType === 'spreadsheet' && this.isTabSelected('Formulas')) {
+		if (docType === 'spreadsheet' && this.isTabSelected('Formula')) {
 			this.updateButtonVisibilityForContext(requestedContext, this.FORMULAS_TAB_ID);
 			return;
 		}
